datafusion-cli/src/main.rs

```diff
@@ -132,6 +179,48 @@ pub async fn main() -> Result<()> {
     Ok(())
 }
 
+fn build_s3_object_store_from_env(bucket_name: &str) -> Result<AmazonS3> {
```

Comment:
https://docs.rs/object_store/latest/object_store/aws/struct.AmazonS3Builder.html#method.from_env

Perhaps?

Reply:
oh haha didn't realize this existed. Thanks!

Reply:
It only exists as of object_store v0.5.0 (cookbook still targets v0.4.0).  

Should update the recipes to the latest version and leverage their features..